### DISCUSSION TITLE
Azure install questions

### DISCUSSION THREAD

**Discussion by peteryang1756** [chatwoot/chatwoot#5334] [Self hosted]
Hello everyone,I try to install on azure,and I got this error,How to fix it?
![Screenshot_20220824-232906_Chrome](https://user-images.githubusercontent.com/92738287/186461005-efeebceb-5a19-4f2b-a58d-1f4f92e3d59c.jpg)

**Comment by sojan-official** [comment]
@ssangyongsports, it's not an error. Installing dependences will take a while.  So you need to wait until it finishes.

**Reply by peteryang1756** [reply]
Is it possible to use azure b1s?

**Reply by sojan-official** [reply]
yes.

**Reply by peteryang1756** [reply]
what wrong?
2022-08-26T02:52:24.195681Z INFO Daemon Agent WALinuxAgent-2.7.3.0 launched with command 'python3 -u bin/WALinuxAgent-2.7.3.0-py2.7.egg -run-exthandlers' is successfully running
